schema/stake_staker.go

```diff
@@ -0,0 +1,14 @@
+package schema
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/shopspring/decimal"
+)
+
+type StakeStaker struct {
+	Address             common.Address  `json:"address"`
+	TotalStakedTokens   decimal.Decimal `json:"total_staked_tokens"`
+	CurrentStakedNodes  uint64          `json:"current_staked_nodes"`
+	CurrentOwnedChips   uint64          `json:"current_owned_chips"`
```

Comment:
use TotalChips, unless we are going to supply a historical count (I don't think so).